main.nf

```diff
@@ -0,0 +1,932 @@
+#!/usr/bin/env nextflow
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+
+========================================================================================
+               NGI - E X O S E Q    B E S T    P R A C T I C E
+========================================================================================
+ #### Homepage / Documentation
+ https://github.com/scilifelab/NGI-ExoSeq
+ #### Authors
+ Senthilkumar Panneerselvam @senthil10 <senthilkumar.panneerselvam@scilifelab.se>
+ Phil Ewels @ewels <phil.ewels@scilifelab.se>
+ Alex Peltzer @alex_peltzer <alexander.peltzer@qbic.uni-tuebingen.de>
+ Marie Gauder <marie.gauder@student.uni-tuebingen.de>
+----------------------------------------------------------------------------------------
+Developed based on GATK's best practise, takes set of FASTQ files and performs:
+ - alignment (BWA)
+ - recalibration (GATK)
+ - realignment (GATK)
+ - variant calling (GATK)
+ - variant evaluation (SnpEff)
+*/
+
+// Package version
+version = '0.8'
+
+// Help message
+helpMessage = """
+===============================================================================
+NGI-ExoSeq : Exome/Targeted sequence capture best practice analysis v${version}
+===============================================================================
+
+Usage: nextflow scilifelab/NGI-ExoSeq --reads '*_R{1,2}.fastq.gz' --genome GRCh37
+
+This is a typical usage where the required parameters (with no defaults) were
+given. The available paramaters are listed below based on category
+
+Required parameters:
+--reads                        Absolute path to project directory
+--genome                       Name of iGenomes reference
+
+Output:
+--outdir                       Path where the results to be saved [Default: './results']
+
+Kit files:
+--kit                          Kit used to prep samples [Default: 'agilent_v5']
+--bait                         Absolute path to bait file
```

Comment:
I think that these are missing from the docs? Maybe they'll be under the todo sections though I guess.

Reply:
I will update these once we have fixed #16 .